packages/lib/shared/utils/chart.helper.ts

```diff
@@ -1,22 +1,81 @@
-export type LabelFormatterParams = {
-  data: any[]
+import { LbpPrice } from '@repo/lib/modules/lbp/pool/usePriceInfo'
```

Comment:
Aren't this functions too specific for LBP to be in a global chart helper file?

Reply:
moved them